test/unit/reporters/CLIReporter-test.js

```diff
@@ -15,13 +15,17 @@ describe('CLIReporter', () => {
   let test = {};
 
   before(() => {
-    loggerStub.transports.console.silent = true;
-    reporterOutputLoggerStub.transports.console.silent = true;
+    [loggerStub, reporterOutputLoggerStub].forEach((logger) => {
+      logger.configure({ silent: true });
```

Comment:
I'm particulraly insterested in how to verify this silent toggling.

Reply:
**Short:** I think it is in the test only to prevent the logger to print to the terminal during the tests. I wouldn't be bothered about it in this phase. If the `--loglevel=silent` option works correctly for Dredd (important), feel free to omit the silencing and leave the logging chatty in tests (nice to have).

**Long:** What I'm trying to achieve is to use winston for Dredd's "application logging" (roughly equal to stderr), and to replace winston completely for the reporters and any other "application output" (roughly equal to stdout). Also, I'd like the logger or the "output rendering component" (not in place yet) to be testable in better ways than it is now. Currently, magic with `proxyquire()` and `sinon.stub()`-s is necessary every time the tests want to verify what was displayed to the user.

The refactoring is hairy and complex, and especially gradually updating the tests to be able to test the same stuff even after changes to the logging infrastructure is tedious. For that reason, I consider having the logger silent during the tests as something _nice to have_ at this moment, and I omitted it many times in my previous PRs. The result is that the output of `npm test` is a bit messy now, but I count with the fact this gets figured out once we have a better way to test the logger (like dependency injection).